rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMModelImpl.java

```diff
@@ -440,7 +440,7 @@ public void setNumNeeded(int pendingTasksCount) {
 
     @Override
     public int getNumNodes() {
-        return numFree + numBusy + numDown;
+        return numFree + numConfiguring + numDeploying + numLost + numBusy + numDown + numToBeRemoved;
```

Comment:
You confirm numToBeRemoved does not include one of the other numXxx?

Reply:
I confirm